src/components/cards/orderModel.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { ShoppingBasket } from 'lucide-react';
+import { createContext } from 'react';
+import { IoClose } from 'react-icons/io5';
+import { Link } from 'react-router-dom';
+import { DynamicData } from '../../@types/DynamicData';
+
+interface OrderModalProps {
+	id: string;
+	idx: number;
+	item: DynamicData;
+	state: DynamicData;
+	handleToggle: (idx: number) => void;
+}
+
+export const modalContext = createContext({});
+export const OrderModel = ({ id, idx, handleToggle }: OrderModalProps) => {
+	return (
+		<div className="absolute desktop:w-[15%] right-2 laptop:right-6 laptop:pr-4 laptop:pt-6 z-50 -top-10 flex p-2 rounded-lg shadow-md bg-[#fcfdfe] _shadow pt-5 pl-5">
+			<div className="flex flex-col justify-between relative w-full">
+				<div className="view_product flex gap-2 text-sm rounded-md p-1 a_link mt-5 mb-3">
+					<Link to={`/orders/${id}`} className="flex gap-2">
+						<ShoppingBasket className="text-sm" /> View More
```

Comment:
improve the design of view more here by centering view more text

Reply:
thx, let me fix it.

Reply:
still i am seeing this issues
![Screenshot 2024-07-20 095058](https://github.com/user-attachments/assets/524cb460-e98b-4e90-a8cb-af4d10f936e5)

Reply:
I solve it.